src/component/ButtonDescription/ButtonDescription.js

```diff
@@ -16,7 +16,9 @@ export default function ButtonDescription({toggleDescription, showDescription, i
           </p>
         )}
       </span>
-      <OpenDescriptionText showDescription={showDescription === id ? true : false}>{description}</OpenDescriptionText>
+      <OpenDescriptionText showDescription={showDescription === id ? true : false}>
```

Comment:
When doing as I suggest, you only need to write:
```suggestion
      <OpenDescriptionText showDescription={showDescription}>
```